packages/lesswrong/server/pageCache/cron.ts

```diff
@@ -0,0 +1,11 @@
+import { addCronJob } from '../cronUtil';
+import PageCacheRepo from '../repos/PageCacheRepo';
+
+addCronJob({
+  name: 'clearExpiredPageCacheEntries',
+  interval: 'every 5 minutes',
+  async job() {
+    const pageCacheRepo = new PageCacheRepo();
+    await pageCacheRepo.clearExpiredEntries();
```

Comment:
@oetherington I experimented with using the trigger approach but it had to run synchronously, which I thought would slow things down unnecessarily